### ISSUE TITLE
sendRawTransaction issue

### ISSUE THREAD

**Issue by slothbag** [ethereum/go-ethereum#2324]
I'm having trouble signing an offline transaction and using sendRawTransaction to submit it.

The tx gets accepted by geth but instead of showing `"to: 0x1234567890"` in the console it shows `"to: &1234567890"`

I'm not sure if the problem is in ethereumjs-tx which creates the raw tx or in geth sendRawTransaction but either way geth should detect the &1234 address and throw error.

See related: https://github.com/ethereumjs/ethereumjs-tx/issues/15

**Comment by karalabe** [comment]
So, we figured out that the issue was an invalid nonce per https://github.com/ethereumjs/ethereumjs-tx/issues/15#issuecomment-195109187 and https://github.com/ethereumjs/ethereumjs-tx/issues/15#issuecomment-195231586. Will close this, please reopen if you find something else that's wrong :)

**Comment by slothbag** [comment]
No problem, I would recommend checking for overflow bugs in Geth though, as it accepts the transaction and shows weird characters in the console.

**Comment by karalabe** [comment]
There's no overflow. The transaction is valid, just misses a "few" ones in between. That is a completely valid use case as might happen when transactions propagate through the network out of order and need to be pieced back together.

Formatting wise we did some internal changes and that probably resulted in printing an address (internally it is one), but I agree it's not the best. @bas-vk How hard would it be / would it make sense to have a custom formatting rule that flattens pointers for the console, specifically the transaction To field which we've changed a few weeks ago?
